src/libraries/pkg/test/packageSettings/System.Management/net6.0/disableVerifyClosure.targets

```diff
@@ -0,0 +1,7 @@
+  <PropertyGroup>
+    <!-- Assembly 'System.Management' is missing dependency 'Microsoft.Win32.Registry'.
+      VerifyClosure doesn't see the Microsoft.Win32.Registry compile time reference as it is
+      being excluded as the library is part of the shared framework. This should go away with
+      https://github.com/dotnet/runtime/issues/53892. -->
+    <ShouldVerifyClosure>false</ShouldVerifyClosure>
```

Comment:
Rather than completely disable this, just exclude Microsoft.Win32.Registry.  Isn't there an item to set to do that?